addons/mail/static/tests/composer/composer.test.js

```diff
@@ -591,6 +602,15 @@ test("quick edit last self-message from UP arrow", async () => {
     await tick();
     await tick();
     await contains(".o-mail-Message .o-mail-Composer", { count: 0 });
+    // ArrowUp for quick edit last stays on last edit message, does not jump to older messages.
```

Comment:
I just put this comment in code, so that these extra-steps make some sense immediately (by themselves they can look random at a glance).